### PR TITLE
[CI] Run stylelint

### LINKED ISSUE #2475: Add stylelint
See https://github.com/Frenzie/FreshRSS/commit/ffb8fc72049583dff25bff0bab04b73931b099a4 for a basic proof of concept. The difficulty/time investment lies in writing down the rules. Hence my question here: https://github.com/FreshRSS/FreshRSS/pull/2245#discussion_r309652495

### PR DESCRIPTION
**Pull request by Frenzie**
Perform some basic CSS sanity checking and style enforcement. Closes #2475.

I removed vendor prefixed linear-gradient and transform because those are from the IE9 era. With IE11 as a minimum and soon to be obsolete requirement it doesn't make much sense anymore.

<hr>

Some files were a bit of a mess indentation-wise (tabs mixed with spaces, inconsistent, etc.) which is why their diffs show up so large. The basic formatting (e.g. a space after `:` or not) can be changed at will by running stylelint with the `--fix` flag.

### CODE REVIEW

File: .stylelintrc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "declaration-colon-space-after": "always",
+    "declaration-colon-space-before": "never",
+    "indentation": "tab",
+    "no-descending-specificity": null,
+    "no-eol-whitespace": true,
+    "property-no-vendor-prefix": true,
+    "rule-empty-line-before": "always",
+    "selector-type-no-unknown": [
+      true, {
+        "ignoreTypes": "as-link",
```
Comment (Frenzie) [inline .stylelintrc]: I thought this was some special SASS thing I didn't know about while I was working on this, but looking at it now I realize it must've been a typo for `.as-link`.

File: p/themes/Ansum/_configuration.scss [hunk 2]
```diff
@@ -1,90 +1,89 @@
...
+		color: $main-font-color;
+		font-size: 3rem;
+		margin-top: 1.75rem;
+		font-weight: 300;
+		line-height: 1.2em;
 	}
-    }
-    
+
+	a[href="./"] { // C'est le bouton "Retour à vos flux"
```
Comment (Frenzie) [inline p/themes/Ansum/_configuration.scss:43]: Origine uses `.header > .item.title h1 a`.

**Comment by Frenzie** [comment]
@aledeg @marienfressinaud Any comments btw? I don't have anything to add here right now.

File: p/themes/Origine/origine.css [hunk 3]
```diff
@@ -697,53 +760,59 @@ a.btn {
...
 .flux.current {
-	border-left: 2px solid #0062BE;
+	background: #fff;
+	border-left: 2px solid #0062be;
 }
+
 .flux.not_read {
-	border-left: 2px solid #FF5300;
-	background: #FFF3ED;
+	background: #fff3ed;
```
Comment (Frenzie) [inline p/themes/Origine/origine.css:785]: @Alkarex Ah, I see the problem. I merged two `.flux.current` selectors but now this one overrides that `#fff`.

Wrt #2604.

I'll fix it, unless you want to.

Comment (Alkarex) [reply p/themes/Origine/origine.css:785]: Go ahead 👍

**Pull request merged**